README.md

```diff
@@ -29,6 +29,8 @@ Here's a quick overview of the major landmarks:
   * [apps/README](./apps/README.md)
   * [blockly/README](https://github.com/code-dot-org/blockly/blob/master/README.md)
 
+For an overview of sites supported by this repository, check out our [URLs by Environment doc](https://docs.google.com/document/d/1rwwOY4l2_zOmO5ZEjTeVStxlIwW_L2iAksWm5FWKxiY).
```

Comment:
The primary audience of the document we’re linking to is internal to our company. Should we explain that in this document to external contributors? Or is there a place we can put this document internally that is easy for people inside our organization to find?

Reply:
Is there a particular reason why "URLs by Environment" needs to be in the shared Google Drive rather than, say, a `.md` file in [`docs/`](https://github.com/code-dot-org/code-dot-org/tree/staging/docs)?`

Reply:
Nope! I asked for a recommendation, and someone told me to put it in Drive. Is there any reason all this information can't be publicly accessible in `docs/`? It feels like it's a better fit for in-repo documentation to me, personally, but I am still figuring out best practice in an open-source codebase.

Reply:
I think Drive is often what we default to, but I agree that in this case keeping it all in the repo makes sense! Certainly I don't see anything in the doc that I think needs to be private.

Reply:
I specifically _removed_ a private Google doc from our in-repo docs [not that long ago](https://github.com/code-dot-org/code-dot-org/pull/49947/commits/f801f96e3c4430df1141577831e88f86fd46aded).

The problem with putting drive links in the repo is that external people will absolutely click on them, and they will also definitely be clicking the "Request Access" button. This triggers an email to anyone with permissions on the doc that asks to share with the external person. All it takes is one person clicking "yes" for the doc to get shared, and that's a lot of noise with and a little bit of risk I'd rather just avoid

The approach I took when I removed the link was to reference the name of the doc, and ensure that the doc was placed in a relatively discoverable place with a clear name in Drive.

Reply:
I would agree that the environment URL details here are safe, and inferrable from other code in the repo. Sticking it in the repo makes sense to me.

Levelbuilder and i18n might be an exception.